templates/pages/admin/entity/notifications.html.twig

```diff
@@ -0,0 +1,205 @@
+         } %}
+      {% endif %}
+      {{ fields.dropdownNumberField('delay_send_emails', item.fields['delay_send_emails'], __('Delay to send email notifications'), {
+         add_field_html: inheritance_labels['delay_send_emails']|default(null),
+         min: 0,
+         max: 100,
+         unit: 'minute',
+         toadd: to_add
+      }) }}
+      {{ _self.inherited_alert_yesno(item, inheritance_labels, 'is_notif_enable_default', __('Enable notifications by default')) }}
+
+      {{ fields.textareaField('mailing_signature', item.fields['mailing_signature'], __('Email signature'), {
+         add_field_html: inheritance_labels['mailing_signature']|default(null),
+      }) }}
+
+      {{ fields.largeTitle(__('Alarms options'), 'ti ti-alarm') }}
+
+      {{ fields.smallTitle('Cartridge'|itemtype_name(get_plural_number())) }}
+      {{ fields.dropdownField('Alert', 'cartridges_alert_repeat', item.fields['cartridges_alert_repeat'], __('Reminders frequency for alarms on cartridges'), {
+         add_field_html: inheritance_labels['cartridges_alert_repeat']|default(null),
+         inherit_parent: item.fields['id'] > 0 ? 1 : 0
+      }) }}
+      {{ fields.dropdownNumberField('default_cartridges_alarm_threshold', item.fields['default_cartridges_alarm_threshold'], __('Default threshold for cartridges count'), {
+         add_field_html: inheritance_labels['default_cartridges_alarm_threshold']|default(null),
+         min: 0,
+         max: 100,
+         toadd: to_add + {
+            (constant('Entity::CONFIG_NEVER')): __('Never')
+         }
+      }) }}
+
+      {{ fields.smallTitle('Consumable'|itemtype_name(get_plural_number())) }}
+      {{ fields.dropdownField('Alert', 'consumables_alert_repeat', item.fields['consumables_alert_repeat'], __('Reminders frequency for alarms on consumables'), {
+         add_field_html: inheritance_labels['consumables_alert_repeat']|default(null),
+         inherit_parent: item.fields['id'] > 0 ? 1 : 0
+      }) }}
+      {{ fields.dropdownNumberField('default_consumables_alarm_threshold', item.fields['default_consumables_alarm_threshold'], __('Default threshold for consumables count'), {
+         add_field_html: inheritance_labels['default_consumables_alarm_threshold']|default(null),
+         min: 0,
+         max: 100,
+         toadd: to_add + {
+            (constant('Entity::CONFIG_NEVER')): __('Never')
+         }
+      }) }}
+
+      {{ fields.smallTitle('Contract'|itemtype_name(get_plural_number())) }}
+      {{ _self.inherited_alert_yesno(item, inheritance_labels, 'use_contracts_alert', __('Alarms on contracts')) }}
+      {{ _self.inherited_alert_dropdown(item, inheritance_labels, 'default_contract_alert', __('Default value'), {
+         0: constant('Dropdown::EMPTY_VALUE'),
+         4: __('End'),
+         8: __('Notice'),
+         12: __('End + Notice'),
+         64: __('Period end'),
+         72: __('Period end + Notice'),
```

Comment:
Instead of hardcoding values, it may be better to pass a `contract_alert_choices` variable that would be generated on PHP side with `'contract_alert_choices' => Contract::getAlertName()`.